src/main/java/org/openhab/binding/zwave/handler/ZWaveSerialHandler.java

```diff
@@ -342,7 +344,7 @@ public void run() {
 
                 }
             } catch (Exception e) {
-                logger.error("Exception during ZWave thread. ", e);
+                logger.debug("Exception during ZWave thread. ", e);
```

Comment:
Probably this should be WARN - this is quite a serious error.

Reply:
Changed to warn.